test/display_test.cpp

```diff
@@ -0,0 +1,75 @@
+#include <cstdint>
+#include <chrono>
+#include <thread>
+#include <string>
+#include <stdexcept>
+
+#include <gtest/gtest.h>
+
+#include "io/Display.hpp"
+
+namespace chip8::test
+{
+    using namespace chip8::io;
```

Comment:
Avoid including namespaces like this (literal namespaces are a exception to this). You only need to add `io::` anyway.

Reply:
Got it